packages/logging/src/v2/index.js

```diff
@@ -15,24 +15,30 @@
  */
 'use strict';
 
-var configServiceV2Client = require('./config_service_v2_client');
 var loggingServiceV2Client = require('./logging_service_v2_client');
+var configServiceV2Client = require('./config_service_v2_client');
 var metricsServiceV2Client = require('./metrics_service_v2_client');
-var extend = require('extend');
 var gax = require('google-gax');
+var extend = require('extend');
+var union = require('lodash.union');
 
 function v2(options) {
   options = extend({
     scopes: v2.ALL_SCOPES
   }, options);
   var gaxGrpc = gax.grpc(options);
   var result = {};
-  extend(result, configServiceV2Client(gaxGrpc));
   extend(result, loggingServiceV2Client(gaxGrpc));
+  extend(result, configServiceV2Client(gaxGrpc));
   extend(result, metricsServiceV2Client(gaxGrpc));
   return result;
 }
 
 v2.SERVICE_ADDRESS = loggingServiceV2Client.SERVICE_ADDRESS;
-v2.ALL_SCOPES = loggingServiceV2Client.ALL_SCOPES;
+v2.ALL_SCOPES = union(
```

Comment:
Same dealio as the other APIs-- all of the `ALL_SCOPES` values are the same, so the `lodash.union` dependency is unnecessary.